### ISSUE TITLE
Add 'height' as a parameter to deplacy.serve function

### ISSUE THREAD

**Issue by James-G-Hill** [KoichiYasuoka/deplacy#2]
Hello,

When running serve with 'port=None' in Jupyter Lab the height of the output cell is currently set at 400 pixels.  That's quite a large gap between cells leaving a lot of whitespace.  I propose a height parameter with default as '400' which would allow the user to specify a smaller (or larger) height if it suits them (200 works best in Jupyter Lab for me).

I'd be willing to create a pull request for this if you would e happy to accept it.

**Comment by KoichiYasuoka** [comment]
Well @James-G-Hill it seems great idea but I'm vague that it is really applicable when default `port=5000`. Umm...

**Comment by James-G-Hill** [comment]
Yeah, sure good point. No worries, I wrote my own custom function anyway.
